roles/foreman_installer_devel_scenario/files/katello-devel-answers.yaml

```diff
@@ -0,0 +1,34 @@
+---
+certs:
+  group: vagrant
+  deploy: true
+  generate: true
+katello_devel:
+  deployment_dir: /home/vagrant
+  user: vagrant
+  db_type: postgres
+  rvm: true
+foreman_proxy_content:
+  pulp_master: true
+  qpid_router_broker_addr: localhost
+foreman_proxy:
+  custom_repo: true
+  http: true
+  ssl_port: '9090'
+  templates: false
+  tftp: false
+  ssl_ca: /etc/foreman-proxy/ssl_ca.pem
+  ssl_cert: /etc/foreman-proxy/ssl_cert.pem
+  ssl_key: /etc/foreman-proxy/ssl_key.pem
+  foreman_ssl_ca: /etc/foreman-proxy/foreman_ssl_ca.pem
+  foreman_ssl_cert: /etc/foreman-proxy/foreman_ssl_cert.pem
+  foreman_ssl_key: /etc/foreman-proxy/foreman_ssl_key.pem
+puppet:
+  server: true
+  server_environments_owner: apache
+  server_foreman_ssl_ca: /etc/pki/katello/puppet/puppet_client_ca.crt
+  server_foreman_ssl_cert: /etc/pki/katello/puppet/puppet_client.crt
+  server_foreman_ssl_key: /etc/pki/katello/puppet/puppet_client.key
+foreman_proxy::plugin::pulp:
```

Comment:
I'm sorry to be the only one who asks for changes, but could you please add more smart proxy plugins here? It's fine to keep them disabled, but at least listed, so they can be enabled via additional installer arguments. Especially if same pattern will be used for luna.

Reply:
@ares Can I do in a follow up? I want to start by porting the functionality as-is to ensure we have a working, known state then do enhancements on top of that state.

Reply:
ok, no problem with that :-)